core/kernel.rbs

```diff
@@ -871,8 +871,8 @@ module Kernel : BasicObject
   #     raise ArgumentError, "No parameters", caller
   #
   def self?.fail: () -> bot
-                | (String arg0) -> bot
-                | (_Exception arg0, ?untyped arg1, ?::Array[String] arg2) -> bot
+                | (String arg0, ?cause: Exception?) -> bot
```

Comment:
> __[Goodcheck]__ Stop using parameter names like `arg0` or `arg1` ([view](https://sider.review/gh/repos/174796313/pulls/1026#issue-6975078))
>
> <details>
>
> The parameter names like `arg0` or `arg1` is not informative enough.
> Try finding good parameter names from documents or arglists.
> If you cannot find a good name, just delete the name of the parameters.
>
> | Rule |
> | ---- |
> | `rbs.no_arg` |
>
> You can [close this issue](https://sider.review/gh/repos/174796313/pulls/1026#issue-6975078) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

Sider suggests two comments for `arg0`, but it is not related to your change. So I'll close the suggestions and merge your PR.

I'll fix the Sider suggestions in another PR.